manifests/install.pp

```diff
@@ -50,5 +57,8 @@
     $teleport::assets_dir:
       ensure => $link_ensure,
       target => "${teleport::extract_path}/teleport/app"
+  } ->
+  file { $teleport_latest_path:
```

Comment:
This adds an extra action to every puppet install, and an extra logline. Is there really much benefit? Personally I see no reason to clean it up as long as its overwritten by `download-teleport-latest'`

Reply:
This action does not serve any specific purpose beside keeping the system clean as that file is not needed (I wasn't aware we wanted to minimise the number of actions and log lines but if that's the case I can just remove it)

Reply:
(removed)

Reply:
I don't think we should minimise log lines, it's trivial to filter logs in any decent client. 

This doesn't mean that we should log junk, duplicates, noise etc. but in general I think debug logging is a very useful feature to have.